pkg/authorization/authorizer/scope/converter.go

```diff
@@ -170,7 +166,11 @@ func DescribeScopes(scopes []string) map[string]string {
 type userEvaluator struct{}
 
 func (userEvaluator) Handles(scope string) bool {
-	return strings.HasPrefix(scope, UserIndicator)
+	switch scope {
+	case UserFull, UserInfo, UserAccessCheck, UserListScopedProjects, UserListAllProjects:
```

Comment:
Probably just easier to check `Validate(scope) == nil` since you are basically binding their logic.

Reply:
@deads2k you probably have an opinion here.